vnfs/libexec/wwmkchroot/functions

```diff
@@ -1,6 +1,12 @@
-eval `wwconfig -as`
+#!/bin/bash
```

Comment:
Unless this PR intends to execute the functions scripts directly, there's no need to add a #! line back into a script that is only ever sourced.

Reply:
Up to you guys if I need to pull this. It's definitely handy with shell checker and any autoformatting editor will handle the file correctly.

Reply:
@bensallen ... I'm not partial either way. For "correctness" it probably isn't needed. But in all honestly, it doesn't hurt anything to have it. Would be nice to hear @jjolly 's thoughts.